primitives/src/lib.rs

```diff
@@ -482,7 +483,7 @@ impl CurrencyId {
 		match self {
 			CurrencyId::Stellar(asset) => asset.decimals(),
 			// We assume that all other assets have 12 decimals
-			CurrencyId::Native | CurrencyId::XCM(_) => 12,
+			CurrencyId::Native | CurrencyId::XCM(_) | CurrencyId::ZenlinkLPToken(_, _, _, _) => 12,
```

Comment:
Strictly speaking this is not true anymore as USDT has now only 6 decimals. (shall we change the XCM definition to extend to 12 decimals instead @ebma?)

Reply:
I think we should, yes. We'd require a migration since the current USDT we have are already bridged using only 6 decimals.
So I guess we can leave this as is.